### PR TITLE
Ability to run subscriptions code snippet as it is

### PR DESCRIPTION
**Pull request by sighclone**
Allows you to copy the snippet of code from this documentation and run it as is, without code changes. The "testing subscriptions" snippet has been updated for the same.

## Description
I have imported the necessary packages and have also created the dependent `subscription` class. I have also defined `schema` in order for the code to run as is.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Documentation

## Issues Fixed or Closed by This PR
#1294 
*

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/operations/testing.md [hunk 1]
```diff
@@ -102,6 +102,20 @@ async def test_mutaton():
 And finally, a test for our [`count` Subscription](docs/general/subscriptions.md):
 
 ```python
+import asyncio
+import pytest as pytest
```
Comment (patrick91) [inline docs/operations/testing.md:106]:
```suggestion
import pytest
```

File: docs/operations/testing.md [hunk 2]
```diff
@@ -102,6 +102,20 @@ async def test_mutaton():
...
+
+@strawberry.type
+class Subscription:
+    @strawberry.subscription
+    async def count(self, target: int = 100) -> int:
+        for i in range(target):
+            yield i
+            await asyncio.sleep(0.5)
+
+schema = strawberry.Schema(query=Query, subscription=Subscription)
```
Comment (patrick91) [inline docs/operations/testing.md:117]: We also need to define query :)

File: docs/operations/testing.md [hunk 3]
```diff
@@ -118,6 +132,7 @@ async def test_subscription():
         assert result.data == {"count": index}
 
         index += 1
+
```
Comment (patrick91) [inline docs/operations/testing.md:135]:
```suggestion
```

**Review by patrick91 — COMMENTED** [review]
Nice one! I've asked for a couple of changes, but it's great overall 😊

File: docs/operations/testing.md (hunk 1 above)
Comment (sighclone) [reply docs/operations/testing.md:106]: will do!

File: docs/operations/testing.md (hunk 2 above)
Comment (sighclone) [reply docs/operations/testing.md:117]: okay gonna do it asap!

File: docs/operations/testing.md (hunk 3 above)
Comment (sighclone) [reply docs/operations/testing.md]: got it

**Comment by sighclone** [comment]
Done! I've made the requested changes

**Review by patrick91 — APPROVED** [review]
Nice one! Thank you 😊

**Comment by botberry** [comment]
Thanks for contributing to Strawberry! 🎉 You've been invited to join
the Strawberry GraphQL organisation 😊

You can also request a free sticker by filling this form: https://forms.gle/dmnfQUPoY5gZbVT67

And don't forget to join our discord server: https://strawberry.rocks/discord 🔥

**Pull request merged**